### DISCUSSION TITLE
[Question] is there any way to get android id or uuid for ios devices to uniquely identify users

### DISCUSSION THREAD

**Discussion by plugeit** [flet-dev/flet#1935] [Q&A] [answered]
### Question

in dart we can get then device uuid (ios) or android id which is uinque to each and every device i am in a very big problem i identify the user based on their page.client_user_agent and page.client_ip but if two users are on same wifi network and are using same browser then i can't distinguish between two users and that is not good. But dart provides a way  to handle this issue i have pasted the sample code for it is there any way to implement it in flet @ndonkoHenri @FeodorFitsner please help

### Code sample

```python
import 'package:device_info/device_info.dart';

String deviceId;

DeviceInfoPlugin deviceInfo = DeviceInfoPlugin();

if (Platform.isAndroid) {
  AndroidDeviceInfo androidInfo = await deviceInfo.androidInfo;
  deviceId = androidInfo.androidId; 
} else if (Platform.isIOS) {
  IosDeviceInfo iosInfo = await deviceInfo.iosInfo;
  deviceId = iosInfo.identifierForVendor;
}
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Feodor will perhaps have to bring it to Flet.

**Reply by FeodorFitsner** [reply, marked as answer]
Looks like a good feature request: https://pub.dev/packages/device_info_plus

**Reply by FeodorFitsner** [reply]
https://github.com/flet-dev/flet/issues/1938
